packages/@aws-cdk/aws-ec2/lib/vpc-lookup.ts

```diff
@@ -55,4 +55,11 @@ export interface VpcLookupOptions {
    * @default Current stack's environment region
    */
   readonly region?: string;
+
+  /**
+   * The ID of the AWS account that owns the VPC
+   *
+   * @default the account id of the parent stack
+   */
+  readonly ownerId?: string;
```

Comment:
I'm wondering if `ownerAccountId` here would be more clear or something like that?

Reply:
Maybe! I went with `ownerId` naively because it's the most direct translation of the underlying filter name, and it allowed for more consistency in the diff. I will rename to the `VpcLookupOptions`'s `ownerAccountId` in a separate commit on this PR.

The internal/private interface (`VpcContextResponse`) might be better kept as `ownerId` to keep the same name as in the actual `DescribeVpcs` call and since it's not exposed to end-users? But happy to fix that one as well.